src/v/cluster/partition_balancer_backend.cc

```diff
@@ -121,9 +122,17 @@ ss::future<> partition_balancer_backend::do_tick() {
         co_return;
     }
 
-    auto health_report
-      = co_await _health_monitor.get_current_cluster_health_snapshot(
```

Comment:
I wonder what should we do with `get_current_cluster_health_snapshot`? Looks like it can return arbitrarily old snapshot which is not very useful. Also, looks like it became unused, maybe we should just delete it?